{"name": "Позитивная проверка получения всех заказов юзера под авторизованным аккаунтом", "status": "passed", "description": "Запрос должен вернуть ошибку и правильный код ответа", "start": 1723366569189, "stop": 1723366569662, "uuid": "e8db738d-cf6f-48b6-92f2-5f0cc9917b46", "historyId": "3a15851f19a3e31c0010f502a182e566", "testCaseId": "3a15851f19a3e31c0010f502a182e566", "fullName": "tests.test_get_concrete_user's_orders.test_get_concrete_user's_orders_without_authorization.TestGetUsersOrders#test_get_users_orders_without_authorization_error", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('reg')"}, {"name": "parentSuite", "value": "tests.test_get_concrete_user's_orders"}, {"name": "suite", "value": "test_get_concrete_user's_orders_without_authorization"}, {"name": "subSuite", "value": "TestGetUsersOrders"}, {"name": "host", "value": "DESKTOP-IKUPUFD"}, {"name": "thread", "value": "16804-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_concrete_user's_orders.test_get_concrete_user's_orders_without_authorization"}]}